docs/contribution/contribution.md

```diff
@@ -0,0 +1,140 @@
+- **Do not** branch off another branch.
+- **Do not** include unrelated changes in the same PR.
+- **Do not** create one massive PR if it can be broken up into smaller PRs.
+
+### What projects can I contribute to?
+
+- [`carbon-components`](https://github.com/carbon-design-system/carbon/tree/master/packages/components)
+- [`carbon-components-react`](https://github.com/carbon-design-system/carbon/tree/master/packages/react)
+- [`carbon-website`](https://github.com/carbon-design-system/carbon-website)
+- [`gatsby-theme-carbon`](https://github.com/carbon-design-system/gatsby-theme-carbon)
+- [`carbon-design-kit`](https://github.com/carbon-design-system/carbon-design-kit)
+- [`@carbon/elements`](https://github.com/carbon-design-system/carbon/tree/master/packages/elements)
+- [`@carbon/type`](https://github.com/carbon-design-system/carbon/tree/master/packages/type)
+- [`@carbon/colors`](https://github.com/carbon-design-system/carbon/tree/master/packages/colors)
+- [`@carbon/grid`](https://github.com/carbon-design-system/carbon/tree/master/packages/grid)
+- [`@carbon/icons`](https://github.com/carbon-design-system/carbon/tree/master/packages/icons)
+- [`@carbon/layout`](https://github.com/carbon-design-system/carbon/tree/master/packages/layout)
+- [`@carbon/motion`](https://github.com/carbon-design-system/carbon/tree/master/packages/motion)
+- [`@carbon/themes`](https://github.com/carbon-design-system/carbon/tree/master/packages/themes)
+- [`@carbon/icons-react`](https://github.com/carbon-design-system/carbon/tree/master/packages/icons-react)
+
+And more! You can view a complete list of our packages
+[here](https://github.com/carbon-design-system/carbon/tree/master/packages).
+
+### Besides some of the obvious contributions mentioned above, how else can I contribute?
+
+Great question! Contribution does not require creating or maintaining our
+assets. Here are some other ways you can contribute, which don't require much
+work:
+
+- **Join our slack community and interact with other users.** We have hundreds
+  of users world wide, and quite a small team in comparison. While we try our
+  best to answer questions on slack, it's not always possible to answer
+  everything. One of the easiest ways to help us it to jump in on slack
+  conversations if there's something you know the answer to! Some of our most
+  popular slack channels include `#carbon-components`, `#carbon-react`,
+  `#carbon-design-system`, `#carbon-ng`, `#carbon-vue`, `#carbon-announcements`,
+  `#carbon-community-components`.
+- **Report bugs.** Even if you don't have the time to contribute a bug fix,
+  opening an issue alone makes a big difference! Be sure to completely fill out
+  the issue template to best help us understand what is going wrong.
+
+### What if I want to contribute a new feature or custom component?
+
+If you're contributing to Carbon Components, we will only accept bug fixes. If
+you want to contribute a custom component, new feature, or a variation of an
```

Comment:
A lot of times to add a feature/variation of a core component it's duplicating a lot of work from the core repo to achieve the desired outcome vs just making the core component more flexible. Would there be a way to denote that a variation in core is owned by someone outside of the core team.  Examples we have: required modals that remove the x on the modal, overflow menus that have headings within the list, etc.